build.sh

```diff
@@ -13,16 +13,107 @@
 # the devfiles in the registry will instead use the value in ./VERSION.
 #
 
+#!/bin/bash
```

Comment:
seems an odd place to put a sh-bang. shouldn't this be on line 1? Oh, wait there's already one there. Not sure what's going on here.... but -1.

Reply:
Yup, copy paste error. Fixed.